main.tf

```diff
@@ -1,4 +1,13 @@
 
+terraform {
+    required_version = ">= 0.12"
+}
+
+provider "google" {
+   credentials = "${file(var.credentials)}"
+   project = var.project_id
+}
+
 data "google_compute_image" "YugaByte_DB_Image" {
   family  = "centos-6"
```

Comment:
I think we should do this test on a few OS's - such as ad Ubuntu and others. We can not assume the end-user will always use Ubuntu. I think Github action allows you to run the same workflow for a combination of OS - I would suggest exploring that.